crawler/src/main/java/com/movie/recommender/crawler/service/MovieService.java

```diff
@@ -1,18 +1,89 @@
 package com.movie.recommender.crawler.service;
 
 import com.movie.recommender.common.client.LocationServiceClient;
+
+import com.movie.recommender.common.model.movie.MovieCredits;
+import com.movie.recommender.common.model.movie.MovieDetails;
+import com.movie.recommender.common.model.movie.MovieKeywords;
+import com.movie.recommender.common.model.movie.NowPlayingMoviesByCountry;
+import com.movie.recommender.crawler.client.SerpApiClient;
+import com.movie.recommender.crawler.client.TmdbApiClient;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
 
+import java.util.*;
+
 @Service
 @Slf4j
 public class MovieService {
     private final LocationServiceClient locationServiceClient;
+    private final TmdbApiClient tmdbApiClient;
+    private final SerpApiClient serpApiClient;
+    private final MongoDBService mongoDBService;
+    private final String DEFAULT_COUNTRY = "ua";
 
-    public MovieService(LocationServiceClient locationServiceClient) {
+    public MovieService(LocationServiceClient locationServiceClient, MongoDBService mongoDBService) {
         this.locationServiceClient = locationServiceClient;
+        this.tmdbApiClient = new TmdbApiClient();
+        this.serpApiClient = new SerpApiClient();
+        this.mongoDBService = mongoDBService;
+    }
+
+    public void loadNowPlayingMovies() {
```

Comment:
loading now playing movies should be done for each country returned by `locationServiceClient`
since there is currently no country code in `countryWithCitiesDTO`, create a hardcoded mapping between country name and country code 
```
Map<String, String> countryCodeMapping = Map.of(
            "Ukraine", "ua" // Hardcoded mapping for now
    );
```
and then handle all countries 
```
List<CountryWithCitiesDTO> countries = locationServiceClient.getAllCountriesAndCities();
for (CountryWithCitiesDTO countryWithCities : countries) {
// here goes your logic with using countryCodeMapping
// ......
    String countryCode = countryCodeMapping.get(countryName);
    
    if (countryCode == null) {
        continue;
    }
//.....
```